pkg/rhcos/builds.go

```diff
@@ -42,3 +45,12 @@ func fetchRHCOSBuild(ctx context.Context) (*metadata, error) {
 
 	return meta, nil
 }
+
+// FetchRHCOSVersion retrives the pinned RHCOS version.
+func FetchRHCOSVersion() (string, error) {
```

Comment:
would `Version` be enough as package is already rhcos?

Reply:
Fixed.